pkgs/development/libraries/boost/1.65.nix

```diff
@@ -9,6 +9,4 @@ callPackage ./generic.nix (args // rec {
     sha256 = "9807a5d16566c57fd74fb522764e0b134a8bbe6b6e8967b83afefd30dcd3be81";
   };
 
-  enableNumpy = args.enableNumpy or true;
```

Comment:
`enableNumpy = args.enableNumpy or `stdenv.hostPlatform == stdenv.buildPlatform;`

Reply:
Yes could do this but it is not ideal: the same  logic is repeated in the generic.nix and for each version. And the proper logic would also take into account if enablePython is passed into args: if that is false, enableNumpy should be false too. 

Ideally it could all be handled in generic.nix?